src/widget/wmainmenubar.cpp

```diff
@@ -519,6 +517,20 @@ void WMainMenuBar::initialize() {
 
     QString externalLinkSuffix = " =>";
 
+    QString keywheelTitle = tr("Show Keywheel");
+    QString keywheelText = tr("Show keywheel");
```

Comment:
This is a translator trap, because they cannot know which string is displayed where.
Please add transla tor comment. //: